discovery-provider/src/tasks/index.py

```diff
@@ -392,9 +392,28 @@ def revert_blocks(self, db, revert_blocks_list):
             session.execute("REFRESH MATERIALIZED VIEW user_lexeme_dict")
 
 
-######## CELERY TASKS ########
+######## IPFS PEER REFRESH ########
+def get_cnode_multiaddrs(db):
+    with db.scoped_session() as session:
+        cnode_endpoints = {}
+        distinct_cnode_endpts = (
+            session.query(User.creator_node_endpoint).filter(User.creator_node_endpoint != None).distinct()
```

Comment:
@hareeshnagaraj so this basically looks for every non None creator_node_enpoint and peers with all of them every time an index job is about to happen?

Reply:
exactly